gnd/src/main/java/com/google/android/gnd/persistence/remote/firestore/DataStoreException.java

```diff
@@ -16,8 +16,26 @@
 
 package com.google.android.gnd.persistence.remote.firestore;
 
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import java8.util.Optional;
+
 public class DataStoreException extends RuntimeException {
   public DataStoreException(String message) {
     super(message);
   }
+
+  @NonNull
+  public static <T> T checkNotNull(@Nullable T o, String field) throws DataStoreException {
```

Comment:
Can you please rename the parameter to something more self-explanatory? `o` is quite ambiguous.

Reply:
Done!